drivers/sensor/bmp180/Kconfig

```diff
@@ -0,0 +1,29 @@
+# BMP180 temperature and pressure sensor configuration options
+
+# Copyright (c) 2016 Intel Corporation
+# Copyright (c) 2017 IpTronix S.r.l.
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig BMP180
+	bool "BMP180 sensor"
+	depends on I2C
+	help
+	  Enable driver for BMP180 I2C-based temperature and pressure sensor.
+if BMP180
+
+choice
+	prompt "BMP180 pressure oversampling"
+	default BMP180_PRESS_OVER_8X
+	help
+	  Select pressure oversampling for the BMP180 sensor.  Higher values
+	  lead to more accurate readings, but higher power consumption.
+config BMP180_PRESS_OVER_1X
+	bool "single"
+config BMP180_PRESS_OVER_2X
+	bool "x2"
+config BMP180_PRESS_OVER_4X
+	bool "x4"
+config BMP180_PRESS_OVER_8X
+	bool "x8"
```

Comment:
This might be better as a devicetree property, as that would allow different values for each driver instance

Reply:
it has a sense, agree. will change it

Reply:
> This might be better as a devicetree property, as that would allow different values for each driver instance

done. now it's moved to DT property